app/styles/components/_map-tools.scss

```diff
@@ -11,21 +11,45 @@
 	.leaflet-control-map-expand {
 		cursor: pointer;
 	}
+
+    .leaflet-control.leaflet-draw {
```

Comment:
Oh, this was left over from when I was using the control, which I'm not doing anymore. Will remove.